Dockerfile.ci-test

```diff
@@ -1,4 +1,6 @@
-FROM golang:1.19 AS builder
+FROM --platform=$BUILDPLATFORM golang:1.19 AS builder
+
+ARG TARGETOS TARGETARCH
```

Comment:
I get a build error from these lines using podman, I think it is expecting a separate line per ARG definition.

Reply:
Updated.
I was able to manually build and publish multiarch images with podman, but the commands
differ a bit from docker.